core/src/processing/core/PApplet.java

```diff
@@ -3867,7 +3867,7 @@ static public final float min(float[] list) {
   }
 
 
-  /**
```

Comment:
Sorry – I was aware of the policy; I just thought you'd rather not receive a one-byte pull request for this. I'm more than happy to submit one if you want it.